youtube_dl/extractor/heise.py

```diff
@@ -40,19 +54,26 @@ def _real_extract(self, url):
         video_id = self._match_id(url)
         webpage = self._download_webpage(url, video_id)
 
+        title = self._html_search_meta('fulltitle', webpage, default=None)
+        if not title or title == "c't":
+            title = self._search_regex(
+                    r'<div[^>]+class="videoplayerjw"[^>]+data-title="([^"]+)"',
+                    webpage, 'title')
+
+        yt_videos = re.findall(
+                r'<iframe[^>]+class="yt_video"[^>]+src="//([^"]+)', webpage)
+        if yt_videos:
+            for i in range(len(yt_videos)):
+                yt_videos[i] = 'https://' + yt_videos[i] 
+            return self.playlist_from_matches(yt_videos, title, 'Youtube')
```

Comment:
This should be rewritten in terms of [`YoutubeIE._extract_urls`](https://github.com/rg3/youtube-dl/commit/66c9fa36c10860b380806b9de48f38d628289e03).